tests/test_entity_meta.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+
+from labelbox.orm.model import Relationship
+from labelbox.orm.db_object import DbObject
+
+
+def test_illegal_cache_cond1():
+
+    class TestEntityA(DbObject):
+        test_entity_b = Relationship.ToOne("TestEntityB", cache=True)
+
+    with pytest.raises(TypeError) as exc_info:
+
+        class TestEntityB(DbObject):
+            another_entity = Relationship.ToOne("AnotherEntity", cache=True)
+
+    assert str(exc_info.value) == \
+        "Cannot cache a relationship to an Entity with its own cached relationship(s)." \
+        " `test_entity_a` caches `test_entity_b` which caches `['another_entity']`"
```

Comment:
More of a nit, but I wouldn't assert for specific error msg - it gets annoying if you have more tests and want to update the message. Also error msg is not a part of API

Reply:
Ok good thought! I'll update it.